phpBB/includes/ucp/ucp_pm_compose.php

```diff
@@ -665,12 +665,6 @@ function compose_pm($id, $mode, $action, $user_folders = array())
 		$subject = (!$subject && $action != 'post') ? $user->lang['NEW_MESSAGE'] : $subject;
 		$message = $request->variable('message', '', true);
```

Comment:
Why are you removing this?

Reply:
I stuffed up.  My ticket ended up on a detached head, not a branch, and it's ended up with unwanted merge commits. I was trying to fix the merge commit and accidentally reverted it.  I'm still new to Git and finding it so confusing. My apologies.

Reply:
You can try 

`git reset --hard HEAD~1`
then
`git push -f origin ticket/16329`

to reset the last commit.

Reply:
Thanks. I think this has fixed it.